migen/lxbuildenv.py

```diff
@@ -0,0 +1,606 @@
+#!/usr/bin/env python3
+
+# This script enables easy, cross-platform building without the need
+# to install third-party Python modules.
+
+LXBUILDENV_VERSION = '2019.8.19.1'
+import sys
+import os
+import subprocess
+import argparse
+
+
+DEPS_DIR = "deps"
+
+DEFAULT_DEPS = {
+    'migen':        'https://github.com/m-labs/migen.git',
```

Comment:
Can you actually leave `litex`, `litedram` and other dependencies blank / unneeded? @xobs?

Reply:
Yes, this is just a list of submodules that get added to `.gitmodules` when you do `python lxbuildenv.py -i`.  Since this command will not be run, this variable won't be checked.

Reply:
i've just made a 1:1 copy of lxbuildenv.py from ../litex and assumed that it's the default version of the script. i didn't invest lxbuildenv so far - is it the idea that the script itself is adjusted for every project? if so, i'd happily remove the unneeded deps...

i was also not particularly happy with just copying it - maybe it would make more sense to add lxbuildenv globally somewhere in the workshop and reference it from there?